src/components/CreateProfile/WorkInformation/WorkInformation.tsx

```diff
@@ -0,0 +1,119 @@
+'use client'
+import React, { useState } from 'react'
+import styles from './WorkInformations.module.scss'
+import { DropdownBio } from '@/inputs/Dropdowns/DropdownBio/DropdownBio'
+import TextArea from '@/inputs/TextArea/TextArea'
+import CheckboxInput from '@/inputs/Checkbox/Checkbox'
+
+const WorkInformation = () => {
+  const initialState = {
```

Comment:
**nitpick:** We should move it out from component body, and type it properly (type object, don't do `as` something)